variables.ini.dist

```diff
@@ -1,147 +1,146 @@
-DATABASE_NAME = ; atmosphere_db
-DATABASE_ENGINE = ; django.db.backends.postgresql_psycopg2
-DATABASE_USER = ; psql_user
-DATABASE_CONN_MAX_AGE = ; 60
-DATABASE_PASSWORD = ; psql_password
-DATABASE_HOST = ; localhost
-DATABASE_PORT = ; 5432
-DJANGO_DEBUG = ; True
-ENFORCING = ; False ;NOTE: DO NOT SET TO TRUE UNLESS YOU ARE PRODUCTION!
-DJANGO_JENKINS = ; False
-DJANGO_TEMPLATE_DEBUG = ; True
-LOG_LIBCLOUD = ; False
-LOGGING_LEVEL = ; logging.INFO
-LOGGING_DEP_LEVEL = ; logging.WARNING
-LOGSTASH_HOST = ; logstash.server.com
-LOGSTASH_PORT = ; 8000
-METRIC_SERVER = ; atmosphere-metrics.server.com
-SELF_SIGNED_CERT = ; False
-TEST_RUNNER = ; custom.test_runner
-TEST_RUNNER_USER = ; test_user
-TEST_RUNNER_PASS = ; test_secret
-MOCK_USER = ; test_user
-USE_MOCK_AUTH = ; False
+AUTH_USE_OVERRIDE =  True # Boolean required will override authentication set in __init__.py
+AUTH_MOCK_USER = # testuser
+AUTH_ENABLE_MOCK =  True # Boolean required
+AUTH_ENABLE_GLOBUS =  False # Boolean required
+AUTH_ENABLE_CAS =  False # Boolean required
+AUTH_ENABLE_LDAP =  False # Boolean required
+AUTH_ENABLE_JWT =  False # Boolean required
+AUTH_ENABLE_OPENSTACK =  False # Boolean required
+ANSIBLE_ROOT = # /path/to/atmosphere-ansible
+LOGOUT_REDIRECT_URL = # /logout
+CAS_AUTH_PREFIX = # /
+CAS_SERVER = # https://cas.server.com
+LDAP_SERVER = # ldap://my.ldap.com
+LDAP_SERVER_DN = # ou=example,dc=org
+JWT_INSTALL =  False # Boolean required
+JWT_PUBLIC_KEY = # /path/to/b64_key.pub
+JWT_CLIENT_KEY = # client_key
+JWT_CLIENT_SECRET = # client_secret
+GLOBUS_OAUTH_ID = # globus_key
+GLOBUS_OAUTH_SECRET = # globus_secret
+GLOBUS_OAUTH_CREDENTIALS_SCOPE = # scope_two
+GLOBUS_OAUTH_ATMOSPHERE_SCOPE = # scope_three
+GLOBUS_TOKENINFO_URL = # https://globus.server.org/token_details
+GLOBUS_TOKEN_URL = # https://globus.server.org/token
+GLOBUS_AUTH_URL = # https://globus.server.org/authorize
+OAUTH_CLIENT_KEY = # oauth_key
+OAUTH_CLIENT_SECRET = # oauth_secret
+KEYSTONE_SERVER = # "https://my-openstack.cloud.org:5000"
+KEYSTONE_DOMAIN_NAME = # "default"
+DATABASE_NAME = atmosphere_db # postgres db name
+DATABASE_ENGINE = django.db.backends.postgresql_psycopg2 # postgres only (for now)
+DATABASE_USER = atmosphere_db_user # psql_user
+DATABASE_PASSWORD = atmosphere_db_pass # psql_password
+DATABASE_HOST = localhost # remote.postgres-server.com
+DATABASE_PORT = 5432
+DATABASE_CONN_MAX_AGE = 60 # in seconds
+DJANGO_DEBUG =  True # Boolean required
+ENFORCING =  False # Boolean required #NOTE: DO NOT SET TO TRUE UNLESS YOU ARE PRODUCTION!
+SSLSERVER =  False # Boolean required
+DJANGO_TEMPLATE_DEBUG =  True # Boolean required
+LOG_LIBCLOUD =  False # Boolean required
+LOGGING_LEVEL = logging.INFO # DEBUG, INFO, WARNING, ERROR
+LOGGING_DEP_LEVEL = logging.WARNING # INFO, WARNING, ERROR
+LOGSTASH_HOST = # logstash.server.com
+LOGSTASH_PORT = # 8000
+METRIC_SERVER = # atmosphere-metrics.server.com
+SELF_SIGNED_CERT =  False # Boolean required
+TESTING =  False # Boolean required
```

Comment:
Yes. But I think I have to look into this just in case things might be broken. I don't think we are using these variables at all. But I am planning to add them back first.